### PR TITLE
View student

### PR DESCRIPTION
**Pull request by J8cob1**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -61,5 +62,13 @@ def get_rm():
                  view_func=RemoveTeam.as_view('removeTeam'),
                  methods=['GET', 'POST'])
 
+app.add_url_rule('/viewStudent/',
+                view_func=ViewStudent.as_view('viewStudent'),
+                methods=['GET'])
+
+app.add_url_rule('/viewStudent/',
```
Comment (necrobuffalo) [inline app.py]: Looks like this was accidentally duplicated?

File: gbmodel/model_sqlalchemy.py [hunk 2]
```diff
@@ -58,6 +58,10 @@ def remove_team(self, name, session_id):
         engine.execute("delete from teams where id = :tid AND session_id = :session_id", params)
         return True
 
+    def getTeamNameFromID(self, team_id):
```
Comment (necrobuffalo) [inline gbmodel/model_sqlalchemy.py]: Per PEP8 function names should use snake_case

**Review by necrobuffalo — CHANGES_REQUESTED** [review]
Looks good for the most part but remember to follow the style guide.

**Comment by J8cob1** [comment]
So me undrafting this pull request was a mistake (it wasn't ready yet), but thank you for the review anyway.

**Pull request closed without merging**